### PR TITLE
Tutorial rework - Blockchain

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Reworks the blockchain tutorial

# Links

- Partial: https://github.com/timescale/docs-private/issues/88

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: tutorials/blockchain-analyze/analyze-blockchain-query.md [hunk 1]
```diff
@@ -0,0 +1,549 @@
...
+</Procedure>
+
+## Do more transactions in a block mean the block is more expensive to mine?
+
+The number of transactions in a block can influence the overall block mining
+fee. For this analysis, a larger time frame is required, so increase the
+analyzed time range to 5 days.
+
+If you choose to visualize the query in Grafana, you can see that the more
+transactions in a block, then the higher the mining fee becomes.
```
Comment (Loquacity) [inline tutorials/blockchain-analyze/analyze-blockchain-query.md:255]:
```suggestion
transactions in a block, the higher the mining fee becomes.
```

File: tutorials/blockchain-query/beginner-blockchain-query.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
+---
+title: Query the Bitcoin blockchain - query data
```
Comment (Rajakavitha1) [inline tutorials/blockchain-query/beginner-blockchain-query.md:2]: I get a 403 in the build preview: https://docs-dev.timescale.com/docs-tutorial-blockchain-lana/tutorials/tutorial-blockchain-lana/blockchain-query/blockchain-query/

**Review by Rajakavitha1 — APPROVED** [review]
LGTM!!! Just the page index needs to be fixed

File: tutorials/blockchain-analyze/index.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+on the Bitcoin blockchain. You can use these skills to query any data on a
+blockchain, including other cryptocurrencies, smart contracts, or health data.
+
+## Prerequisites
+
+Before you begin, make sure you have:
+
+*   Signed up for a [free Timescale account][cloud-install].
+*   <Optional />If you want to graph your queries, signed up for a
+    [Grafana account][grafana-setup].
```
Comment (Rajakavitha1) [inline tutorials/blockchain-analyze/index.md:27]:
```suggestion
*   <Optional />Signed up for a [Grafana account][grafana-setup] to graph your queries.
```

File: tutorials/blockchain-analyze/index.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+1.  [Set up up your dataset][blockchain-dataset]
+1.  [Analyze your dataset][blockchain-query]
+
+## About analyzing the Bitcoin blockchain with Timescale
+
+This tutorial uses a sample Bitcoin dataset to show you how to aggregate
+blockchain transaction data, and construct queries to analyze information from
+the aggregations. The queries you do in this tutorial are used to do things like
+determine if a cryptocurrency has a high transaction fee, shows any correlation
+between transaction volumes and fees, or if it's expensive to mine.
```
Comment (Rajakavitha1) [inline tutorials/blockchain-analyze/index.md:42]:
```suggestion
the aggregations. The queries in this tutorial help you
determine if a cryptocurrency has a high transaction fee, shows any correlation
between transaction volumes and fees, or if it's expensive to mine.
```

File: tutorials/blockchain-query/beginner-blockchain-query.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+*   [What are the five most recent transactions?](#what-are-the-five-most-recent-transactions)
+*   [What are the five most recent coinbase transactions?](#what-are-the-five-most-recent-coinbase-transactions)
+*   [What are the five most recent blocks?](#what-are-the-five-most-recent-blocks?)
+
+## What are the five most recent transactions?
+
+This dataset contains Bitcoin transactions for the last five days. To find out
+the most recent transactions in the dataset, you can use a `SELECT` statement.
+In this case, you want to find transactions that are not coinbase transactions,
+order them by time in descending order, and take the top five results. You also
```
Comment (Rajakavitha1) [inline tutorials/blockchain-query/beginner-blockchain-query.md:25]:
```suggestion
sort them by time in descending order, and take the top five results. You also
```

**Review by Rajakavitha1 — APPROVED** [review]
LGTM!!!

**Pull request merged**